SkierScribbler/src/Interface/WebView/WebViewBridge.swift

```diff
@@ -48,8 +49,16 @@ struct WebViewBridge: UIViewRepresentable {
 
     func updateUIView(_ uiView: WKWebView, context: Context) {
         let request = URLRequest(url: url)
-
-        uiView.load(request)
+        
+        if UIApplication.shared.applicationState == .active {
+            if url.isArticleURL {
+                uiView.load(request)
+            } else {
+                Sheet(isPresented: .constant(true)) {
+                    WebViewBridge(url: .constant(url), percentLoaded: $percentLoaded, isDoneLoading: $isDoneLoading)
+                }
+            }
+        }
```

Comment:
Remove this! The original was good. We don't need any of this stuff - it's all handled in the `ArticleView`. However, we *do* need to update `WebViewBridge` so it updates the URL when it handles whether or not to navigate - that could also be in a seperate PR. Without that it won't work. So remove this, and in `decideActionFor` in the `Coordinator`, update the method to update the `@Binding` url with the new url in the function.